### PR TITLE
Move code review documentation to markdown

### PR DESCRIPTION
**Pull request by Onager**

### CODE REVIEW

File: process/Code review process.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Therefore we would like to ask people to hang on, to get through the code review process and try to learn something while going through it. Rest assured, it will get easier next time and even easier the time after that, and before you know it you can contribute code to the project with little to no comments.
+
+And if things are unclear, don't hesitate to ask. The developer mailing list is: log2timeline-dev@googlegroups.com or you can join the slack channel [here](https://t.co/mO1re10Bfs). 
+
+## Process
```
Comment (joachimmetz) [inline process/Code review process.md]: Maybe rename the heading to Detailed steps seeing the previous heading is already named Process

File: process/Code review process.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+First time contributors may come across the fact that the code review process can take quite a long time, with lots of back and forth comments. You may think that you are wasting the core developers time, but rest assured you are not. We look at this as an investment of building up good solid code contributors. We want to make sure our contributors understand the code and the style guide and will make suggestions to the contributor to fix what we think needs improving. Despite spending potentially more time to begin with to get code submitted into the project we believe this investment in code review will result in better code submissions and increased proficiency of the contributor.
+
+Therefore we would like to ask people to hang on, to get through the code review process and try to learn something while going through it. Rest assured, it will get easier next time and even easier the time after that, and before you know it you can contribute code to the project with little to no comments.
+
+And if things are unclear, don't hesitate to ask. The developer mailing list is: log2timeline-dev@googlegroups.com or you can join the slack channel [here](https://t.co/mO1re10Bfs). 
+
```
Comment (joachimmetz) [inline process/Code review process.md:33]: Maybe point to setting up a dev environment here first?

**Review by joachimmetz — CHANGES_REQUESTED** [review]
Some nits, nothing major

File: process/Code review process.md (hunk 1 above)
Comment (Onager) [reply process/Code review process.md]: Done

File: process/Code review process.md (hunk 2 above)
Comment (Onager) [reply process/Code review process.md:33]: Done

**Review by joachimmetz — APPROVED** [review]

**Pull request merged**